### PR TITLE
updated iteration cnt for validate designs and provide env variable support

### PR DESCRIPTION
**Pull request by vishnuchebrolu**

### CODE REVIEW

File: tests/validate/bandwidth_test/src/host.cpp [hunk 1]
```diff
@@ -124,8 +124,15 @@ int main(int argc, char** argv) {
     }
 
     double max_throughput = 0;
+    char* env_reps_cnt = getenv("XRT_VALIDATE_ITER_CNT");
+    int reps = 10000;
+    if (env_reps_cnt) {
+        reps = atoi(env_reps_cnt);
+        std::cout << "INFO : we found command line argument “XRT_VALIDATE_ITER_CNT”=" << reps
+                  << " and using the same for kernel iteration.\n";
+    }
```
Comment (stsoe) [inline tests/validate/bandwidth_test/src/host.cpp:133]: Should this env var be a command line argument instead?

File: tests/validate/slavebridge_test/src/host.cpp [hunk 2]
```diff
@@ -125,8 +125,15 @@ int main(int argc, char** argv) {
...
 
     double max_throughput = 0;
+    char* env_reps_cnt = getenv("XRT_VALIDATE_ITER_CNT");
+    int reps = 10000;
+    if (env_reps_cnt) {
+        reps = atoi(env_reps_cnt);
+        std::cout << "INFO : we found command line argument “XRT_VALIDATE_ITER_CNT”=" << reps
+                  << " and using the same for kernel iteration.\n";
+    }
+
```
Comment (stsoe) [inline tests/validate/slavebridge_test/src/host.cpp:135]: Ditto

File: tests/validate/bandwidth_test/src/host.cpp (hunk 1 above)
Comment (AShivangi) [reply tests/validate/bandwidth_test/src/host.cpp:133]: I had the same concern so we had a discussion via email. Here is what I got: "environmental variable is intended to be for internal test only and undocumented from a customer perspective". If you're interested in further details, I can add you to the email change.

**Review by AShivangi — APPROVED** [review]

File: tests/validate/bandwidth_test/src/host.cpp (hunk 1 above)
Comment (stsoe) [reply tests/validate/bandwidth_test/src/host.cpp:133]: That's what hidden options are for.    Can we please change?

**Review by stsoe — CHANGES_REQUESTED** [review]

File: tests/validate/bandwidth_test/src/host.cpp (hunk 1 above)
Comment (vishnuchebrolu) [reply tests/validate/bandwidth_test/src/host.cpp]: Updated.

File: tests/validate/slavebridge_test/src/host.cpp (hunk 2 above)
Comment (vishnuchebrolu) [reply tests/validate/slavebridge_test/src/host.cpp]: updated.

**Review by stsoe — APPROVED** [review]

**Pull request merged**